chesster.js

```diff
@@ -610,6 +729,7 @@ function exception_handler(bot, message, todo){
             "\nDatetime: " + new Date() + 
             "\nMessage: " +  JSON.stringify(message) + 
             "\nError: " + JSON.stringify(e) + 
+            "\nStack: " + e.stack +
```

Comment:
maybe I have not been paying attention, but didnt the stringify on the error already show the stack? 

If not, good call. Thanks.